README.adoc

```diff
@@ -6,14 +6,14 @@ Spring Boot helps you to create Spring-powered, production-grade applications an
 It takes an opinionated view of the Spring platform so that new and existing users can quickly get to the bits they need.
 
 You can use Spring Boot to create stand-alone Java applications that can be started using `java -jar` or more traditional WAR deployments.
-We also provide a command line tool that runs Spring scripts.
+We also provide a command-line tool that runs Spring scripts.
 
 Our primary goals are:
 
 * Provide a radically faster and widely accessible getting started experience for all Spring development.
 * Be opinionated, but get out of the way quickly as requirements start to diverge from the defaults.
 * Provide a range of non-functional features common to large classes of projects (for example, embedded servers, security, metrics, health checks, externalized configuration).
-* Absolutely no code generation and no requirement for XML configuration.
+* No code generation and no requirement for XML configuration.
```

Comment:
I'm not sure why absolutely is being removed here.

Reply:
"Absolutely" seemed redundant here but I am changing it back accordingly.